docs/user-guide/agile-boards/README.md

```diff
@@ -151,7 +151,9 @@ Apart from the Status board you can **update a work package's status** directly
 
  ![OpenProject-Boards_update-status](OpenProject-Boards_update-status-1568640175105.png)
 
-A **double click on a card** will open the work package's **fullscreen view.** The **arrow** on top will bring you back to the boards view. ![OpenProject-Boards_card-details](OpenProject-Boards_card-details-1568640191629.png)
+A **double click on a card** will open the work package's **fullscreen view.** The **arrow** on top will bring you back to the boards view. 
+
+![Bildschirmfoto 2022-08-26 um 09.58.31](Bildschirmfoto 2022-08-26 um 09.58.31.png)
```

Comment:
Hi @JanKeppler, please don't forget to add an image description in the [...], e.g. "[Arrow back to boards view]"(filename.png).

Reply:
And please remove the spaces in the filename :)

Reply:
@birthe @as-op Thanks for pointing that out, I must have missed it. I have adjusted it now.